src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Mappers/WhenIMapEditDraftApprenticeshipToBaseDraftApprenticeshipRequest.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AutoFixture;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using FluentAssertions;
+using SFA.DAS.ProviderCommitments.Web.Mappers;
+using SFA.DAS.ProviderCommitments.Web.Models;
+
+namespace SFA.DAS.ProviderCommitments.Web.UnitTests.Mappers
+{
+    [TestFixture]
+    public class WhenIMapEditDraftApprenticeshipToBaseDraftApprenticeshipRequest
+    {
+        private EditDraftApprenticeshipToBaseDraftApprenticeshipRequestMapper _mapper;
+        private EditDraftApprenticeshipViewModel _source;
+
+        [SetUp]
+        public void Arrange()
+        {
+            var fixture = new Fixture();
+            _source = fixture.Build<EditDraftApprenticeshipViewModel>().Without(x => x.BirthDay).Without(x => x.BirthMonth).Without(x => x.BirthYear)
```

Comment:
Is it important that these properties are null?  I can't see any reason for it - but could be missing something!

Reply:
Autofixture, creates them with invalid values, if not excluded